lib/routes/jandan/girl.ts

```diff
@@ -0,0 +1,67 @@
+import { Data, DataItem, Route, ViewType } from '@/types';
+import { Context } from 'hono';
+import ofetch from '@/utils/ofetch';
+import path from 'node:path';
+import { art } from '@/utils/render';
+import { getCurrentPath } from '@/utils/helpers';
+
+export const route: Route = {
+    path: '/girl',
+    name: '煎蛋妹子图',
+    example: '/girl',
+    maintainers: ['lemon'],
+    handler,
+    view: ViewType.Pictures,
+};
+
+async function handler(ctx: Context): Promise<Data> {
+    const limit = ctx.req.query('limit') ? Number.parseInt(ctx.req.query('limit')!) : 30;
+
+    const items = await crawl(ctx, limit);
+
+    return {
+        title: '妹子图 - 煎蛋',
+        link: 'https://jandan.net',
+        item: items,
+    };
+}
+
+async function crawl(ctx: Context, limit: number = 30) {
+    const data = await _crawl(undefined, limit, []);
+    // set debug info
+    ctx.set('json', data);
+    return data;
+}
+
+const __dirname = getCurrentPath(import.meta.url);
+const girlDescTemplate = path.join(__dirname, 'templates/girl.art');
+
+async function _crawl(startId?: string, limit: number = 30, items: DataItem[] = []): Promise<DataItem[]> {
+    if (items.length >= limit) {
```

Comment:
No page turn.

Reply:
what did you mean?

Reply:
https://github.com/DIYgod/RSSHub/blob/60dc249fef0093ed8d20c346d6b9f99436bdfaa6/lib/routes/jandan/girl.ts#L63-L65

Reply:
there are no extra parameters in Jandan's API